LinuxMac.sh

```diff
@@ -0,0 +1,63 @@
+#   mss-install.LinuxMac
+#   ~~~~~~~~~~~~~~~~~~~~~
+#
+#   This script tries to install conda and/or mss on a Linux or MacOS system automatically.
+#
+#   This file is part of mss.
+#
+#   :copyright: Copyright 2021 May Baer
+#   :copyright: Copyright 2021 by the mss team, see AUTHORS.
+#   :license: APACHE-2.0, see LICENSE for details.
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+#!/bin/bash -i
+automatic=$(([ "$1" = "a" ] || [ "$1" = "-a" ]) && echo "Yes" || echo "No")
+echo "Checking Conda installation..."
+condaInstalled="Yes"
+conda || { echo "Downloading miniconda3..." &&
+   condaInstalled="No" &&
+   unameOS=$([ "$(uname -s)" == "Darwin" ] && echo "MacOSX" || echo "Linux") &&
+   curl "https://repo.anaconda.com/miniconda/Miniconda3-latest-${unameOS}-x86_64.sh" --output miniconda-installer.sh &&
+   echo "Installing miniconda3..." &&
+   chmod +x miniconda-installer.sh &&
+   if [[ $unameOS = "MacOSX" ]]
```

Comment:
I'm not familiar with Mac's. I guess MSS does not run on a ARM M1 Mac directly. 
Does those have a different unameOS ?
If MSS can't currently be used there the installation step should explain this early and skip setting up miniconda.

Reply:
ARM M1 is now checked for and causes an early abort.